client/src/components/App.jsx

```diff
@@ -91,15 +119,26 @@ class App extends React.Component {
             averages={this.state.averages}
             reviews={this.state.reviews}
           />
-          <WriteReview />
+          <WriteReview
+            showModal={this.showWriteReviewModal}
+            percentRecommended={this.state.percentRecommended}
+          />
           <ReviewList
             loadAllReviews={this.loadAllReviews}
             loadAll={this.state.loadAll}
             reviews={this.state.reviews}
           />
-          <PhotoGallery reviews={this.state.reviews} showModal={this.showModal}/>
+          <PhotoGallery reviews={this.state.reviews} showModal={this.showPhotosModal}/>
         </StyledApp>
-        <PhotoGalleryModal hideModal={this.hideModal} isVisible={this.state.showPhotos} reviews={this.state.reviews} />
+        <PhotoGalleryModal
+          hideModal={this.hidePhotosModal}
+          isVisible={this.state.showPhotos}
+          reviews={this.state.reviews}
+        />
+        <WriteReviewModal
+          hideModal={this.hideWriteReviewModal}
+          isVisible={this.state.showReviewForm}
```

Comment:
I think you might want to refactor to make these destructured, per AirBnb style guide!